### ISSUE TITLE
input value with double quotes, how to remove?

### ISSUE THREAD

**Issue by yongmin1983** [yairEO/tagify#91]
`<td style="" colspan="0" class="text-left"><tags class="tagify form-control tags text-left ">
                    <tag title="abd" value="abd">
                            <x title=""></x><div><span>abd</span></div>
                        </tag><tag title="dabve" value="dabve">
                            <x title=""></x><div><span>dabve</span></div>
                        </tag><div data-placeholder="" class="tagify__input" contenteditable=""></div>
                </tags><input name="keywords" id="keywords" class="form-control tags text-left" value="" style="display:none;width:100%;"></td>`

I got 

keywords            = rowBox.find('input[name="keywords"]');

value with: "" which cause erros

**Comment by yairEO** [comment]
`value=""` cause errors? I cannot see how

**Comment by me4g00g** [comment]
@yongmin1983 If I get you correctly, the values in the result have double quotes appended to them. If this is the case, simply before submitting the form, use the below code
`keywords = rowBox.find('input[name="keywords"]');
keywords = keywords.replace("/\"/g", "");
`
Hope This helps.

**Comment by yongmin1983** [comment]
@me4g00g @yairEO 

Thanks for your reply.
It does cause error when you send to back end with extra quote.

@me4g00g Your method is what I am using but I think there should add some configuration or seeting to decide with or without Quotes

**Comment by yairEO** [comment]
where did you see an extra quote? how did you get to have it?

The demo page has no problems and I can't seem to reproduce it there

**Comment by yongmin1983** [comment]
`<input name="keywords" id="keywords" class="form-control tags text-left" value="" >`

This is the html code.

`$('.tags').each(function(){
        $elm = $( this ).tagify().on('add', function(e, tagName){
            console.log('added', tagName)
        });
    });`

JS.

when I post： 

keywords | "key"
-- | --

It was expected as 

keywords | key
-- | --

**Comment by philoj** [comment]
@yairEO 
I ran into a double quote issue:
Refreshing a page containing a 'tagified' input with at least one tag already typed into it would cause the tag content to add double quotes on either side with every refresh.
for ex:
add tag 'tag'
refresh
'tag' now becomes '"tag"'
refresh again,
'"tag"' becomes '"\"tag\""'

without the single quotes, of course

**Comment by yairEO** [comment]
It's because the input's value cannot represent tags as text (`String`). 

@jesuismaxime  - does your template re-renders when the user goes back to the "tagify" page?
Do you use any state to save the page's state somewhere?

In order to fix this for now you will need to reset the tagify field and do:

    // Where you actually bind the `tagify` on the input field, you need to reset the input before applying
    inputElm.value = '';
    // Then apply Tagify on the input and then do:
    // assuming you saved `tagify.value` in your state, lets say in `this.state.tags`
    tagify.addTags( this.state.tags )

You can do this in your React lifecycle (assuming you're using React), I think `componentDidUpdate` will work


Also, I am working on a React version of the Tagify, that will be released soon

**Comment by eddieishere** [comment]
Is there an update on this issue yet?

**Comment by yairEO** [comment]
@eddieishere - please describe your problem in details because after re-reading this thread, I am unsure what's the actual problem and how to re-produce

**Comment by eddieishere** [comment]
Well the whitelist produces an array instead of a string, so the brackets, double quotes and value need removed. 

[{"value"%3A"""}]

**Comment by yairEO** [comment]
@eddieishere - sorry but I don't understand your explanation. Please describe the scenario in details. 

How can a whitelist produces anything? it's not a machine. it's a static Array. Please, if you are able, write the steps needed to replicate this.

**Comment by eddieishere** [comment]
The problem is it’s an array and we need a string... I think that is very clear from this thread.

**Comment by yairEO** [comment]
What is the problem? please describe EXACT steps with as **much detail as possible**.

I've asked you 2 times already to describe in detail and you've ignored me, so how am I suppose to help? I honestly do not understand this issue.

**Comment by eddieishere** [comment]
"since nobody else"? You do realise someone else started this thread? That makes at least two people you know, which contradicts your statement. I have **EXPLAINED** enough. So I've ignored you have I? Well, get used to that, because this thread is a total waste of time.

**Comment by yairEO** [comment]
@eddieishere 
I wouldn't have spent my own time trying to help if it was a waste of time, as i'm trying to understand what's going on. If me, the author of this project, cannot understand than who can?

Why can't you simply do as I've asked and explain this in steps? 
You do not want me to help? 
You do not respect my time, making this free software + making free time to help with it?

You've explained nothing so far. I've ask you to write me steps. 1,2,3,4, etc. on how to reproduce this, but you refuse..
